tests/providers/apache/livy/hooks/test_livy.py

```diff
@@ -17,34 +17,55 @@
 from __future__ import annotations
```

Comment:
A lot of changes in this module:

1. Make sure that changes in tests-connections reflected in tests
2. Replace all SubTests
3. replace `requests_mock.mock` decorator by `requests_mock` fixture
4. Add additional test parametrisation